tests/role_wrapper.yml

```diff
@@ -0,0 +1,82 @@
+- name: Create a temporary file for a static inventory
+  tempfile:
+    state: file
+    suffix: .inventory.yaml
```

Comment:
`.yml` not `.yaml`

Reply:
Is this file stored on the developer machine or on the test VM? I guess the developer machine. Then it should be named more clearly and maybe even be a temporary file to avoid conflicts/irritation for developers.

Reply:
yes, developer machine/control host, see `delegate_to: localhost` below

Reply:
It is stored on control node/developer machine as temporary file under `/tmp`.

Reply:
sorry for the confusion, I missed that it is actually using the tempfile module